src/spikeinterface/core/waveform_extractor.py

```diff
@@ -1612,6 +1621,8 @@ def extract_waveforms(
             ms_before=ms_before,
             ms_after=ms_after,
             num_spikes_for_sparsity=num_spikes_for_sparsity,
+            unit_batch_size=unit_batch_size,
+            temp_folder=sparsity_temp_folder,
```

Comment:
This 2 args should be in estimate_kwargs.

Reply:
then `num_spikes_for_sparsity` too? Still good to list them in the docstrings though